Modules/Microsoft365DSC/DSCResources/MSFT_IntuneWifiConfigurationPolicyIOS/MSFT_IntuneWifiConfigurationPolicyIOS.psm1

```diff
@@ -257,7 +266,7 @@ function Set-TargetResource
         $Ssid,
 
         [Parameter()]
-        [ValidateSet('open', 'wpaPersonal', 'wpaEnterprise', 'wep', 'wpa2Personal', 'wpa2Enterprise')]
+        [ValidateSet('open', 'wpaPersonal', 'wep')]
```

Comment:
Removing values from the ValidateSet would be considered as breaking change. Any chance to keep the previous elements and wirte a warning about them being deprecated?